release-notes.md

```diff
@@ -4,6 +4,12 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## April 8, 2021
+
+### iOS AEPEdgeIdentity
+
+The Adobe Experience Platform Edge Identity (AEPEdgeIdentity\) mobile extension is now available in iOS! This extension enables identity management with the Adobe Experience Platform and Adobe Experience Cloud solutions, by leveraging Experience Edge capabilities.
```

Comment:
Should we also mention the Launch extension in here, something simple like:
You can find the `Edge Identity` extension in the Launch extensions catalog for mobile properties.